src/cider/nrepl/middleware/classpath.clj

```diff
@@ -2,10 +2,17 @@
   (:require
    [cider.nrepl.middleware.util.error-handling :refer [with-safe-transport]]
    [clojure.java.io :as io]
-   [orchard.java.classpath :as cp]))
+   [orchard.java.classpath :as cp]
+   [orchard.misc :as misc]))
+
+(defn file-url?
```

Comment:
Btw, doesn't it make more sense for this to be in `orchard` as well? Or at least a private function here?